resources/views/pagination.blade.php

```diff
@@ -80,11 +80,14 @@ class="py-2 px-3 w-full bg-transparent dark:text-theme-secondary-200"
             </form>
 
             <div class="hidden px-2 rounded bg-theme-primary-100 dark:bg-theme-secondary-800 md:flex flex-inline">
+                @php ($pageIndex = 1)
```

Comment:
there's a `$loop->index` available in `foreach` loops

Reply:
yeh but this is the result of 2 loops 🤔

Reply:
e.g. `@php ($pageIndex++)` is inside a second loop

Reply:
I see, alrighty then